src/Findologic/Request/FindologicRequestFactory.php

```diff
@@ -74,6 +50,13 @@ protected function setDefaults(
         $searchNavigationRequest->setUserIp($this->fetchClientIp());
         $searchNavigationRequest->setRevision($this->getPluginVersion());
         $searchNavigationRequest->setOutputAdapter(OutputAdapter::XML_21);
+        $searchNavigationRequest->addIndividualParam('shopType', 'Shopware6', FindoLogicApiRequest::ADD_VALUE);
+        $searchNavigationRequest->addIndividualParam(
+            'shopVersion',
+            Utils::getCleanShopwareVersion(),
+            FindoLogicApiRequest::ADD_VALUE
+        );
```

Comment:
This should not be `ADD_VALUE` but `SET_VALUE`, as this parameter should only be set once and not be able to have multiple values.
```suggestion
        $searchNavigationRequest->addIndividualParam('shopType', 'Shopware6', FindoLogicApiRequest::SET_VALUE);
        $searchNavigationRequest->addIndividualParam(
            'shopVersion',
            Utils::getCleanShopwareVersion(),
            FindoLogicApiRequest::SET_VALUE
        );
```